### PR TITLE
Feature/applied jobs page

### PR DESCRIPTION
**Pull request by yanisurbis**
Hello, while browsing I noticed this function:
```
(defn state->candidate-status
  [s]
  (cond (or (= s :approved)
            (= s :pending))  "Pending"
        (= s :get_in_touch)  "Interviewing \u2728"
        (or (= s :pass)
            (= s :rejected)) "Rejected \uD83D\uDE41"
        (= s :hired)         "Hired"))
```
I'm not sure it's correct

### CODE REVIEW

**Comment by lacogubik** [comment]
Fixes #8

**Comment by lacogubik** [comment]
> Hello, while browsing I noticed this function:
> 
> ```
> (defn state->candidate-status
>   [s]
>   (cond (or (= s :approved)
>             (= s :pending))  "Pending"
>         (= s :get_in_touch)  "Interviewing \u2728"
>         (or (= s :pass)
>             (= s :rejected)) "Rejected \uD83D\uDE41"
>         (= s :hired)         "Hired"))
> ```
> 
> I'm not sure it's correct

@yanisurbis  Why do you think it's incorrect?

**Comment by yanisurbis** [comment]
@lacogubik If I understood correctly if job application will become approved I will still be having `Pending` message

**Comment by lacogubik** [comment]
> @lacogubik If I understood correctly if job application will become approved I will still be having `Pending` message

@yanisurbis That's correct. `approved` state of application means that it's approved for company user to see it, therefore from candidate's point of view it's still pending.

**Comment by yanisurbis** [comment]
@lacogubik Got it, thank you

I noticed that in `applied jobs` section in dashboard `easy apply` buttons were not disabled. The reason is absence of `applied` field. When you click on this button server returns an error.

I tried to fetch jobs with `applied` but got null for this field. Also I couldn't find GraphiQL so this behavior is still present in new page.

photo_2019-10-01_22-05-19

File: client/src/wh/logged_in/personalised_jobs/views.cljs [hunk 1]
```diff
@@ -33,4 +33,5 @@
         :else (case type-of-jobs
                 :recommended [[:p "Add some skills and preffered locations to your profile to see recommendations."]]
                 :liked [[:p "Click on some " [icon "like" :class "like red-fill"] " to save jobs you like."]]
+                :applied [[:p "Applied for some jobs in our " [link "job board" :jobsboard :class "a--underlined"] "."]]
```
Comment (acron0) [inline client/src/wh/logged_in/personalised_jobs/views.cljs]:
```suggestion
                :applied [[:p "You haven't applied for any jobs yet... " [link "What are you waiting for?" :jobsboard :class "a--underlined"] "."]]
```

**Comment by lacogubik** [comment]
> @lacogubik Got it, thank you
> 
> I noticed that in `applied jobs` section in dashboard `easy apply` buttons were not disabled. The reason is absence of `applied` field. When you click on this button server returns an error.
> 
> I tried to fetch jobs with `applied` but got null for this field. Also I couldn't find GraphiQL so this behavior is still present in new page.
> 

`applied` is not fetched from graphql API as part of the job entity, but it's added by [subscription](https://github.com/WorksHub/client/blob/master/client/src/wh/user/subs.cljs#L18).
Liked and applied jobs should be injected into initial app-db when you login.
However there was bug for the dev login and applied and liked jobs were not correctly added. I have fixed that and now you should see apply buttons and also heart icon in correct state.

**Comment by lacogubik** [comment]
OK TO MERGE

**Comment by WorksHubCodi** [comment]
Thanks for contributing! This code has been merged upstream!

**Pull request closed without merging**